dev_docs/shared_ux/chrome_recently_accessed/chrome_recently_accessed.mdx

```diff
@@ -0,0 +1,62 @@
+---
+id: kibDevDocsChromeRecentlyAccessed
+slug: /kibana-dev-docs/chrome/recently-accessed
+title: Chrome Recently Viewed
+description: How to use chrome's recently accessed service to add your links to the recently viewed list in the side navigation.
+date: 2024-10-04
+tags: ['kibana', 'dev', 'contributor', 'chrome', 'navigation', 'shared-ux']
+---
+
+## Introduction
+
+The `ChromeRecentlyAccessed` service allows applications to register recently visited objects. These items are displayed in the "Recently Viewed" section of a side navigation menu, providing users with quick access to their previously visited resources. This service includes methods for adding, retrieving, and subscribing to the recently accessed history.
+
+![Recently viewed section in the sidenav](./chrome_recently_accessed.png)
+
+## Guidelines
+
+The `ChromeRecentlyAccessed` service should be used thoughtfully to provide users with easy access to key resources they've interacted with. Unlike browser history, this feature is for important items that users may want to revisit.
+
+DOs:
+
+- Register important resources that users may want to revisit. Like a dashboard, a saved search, or another specific object.
+- Update the link when the state of the current resource changes. For example, if a user changes the time range while on a dashboard, update the recently viewed link to reflect the latest viewed state where possible.
+
+DON'Ts:
+
+- Don't register every page view.
+- Don't register temporary or transient states. Skip state changes.
```

Comment:
"Skip state changes" feels a little bit contradictory with the guidance to "Update teh link when the stet of the current resource changes."

Maybe it would be less confusing to simplify and change this line to:
```suggestion
- Don't register temporary or transient states as individual items.
```